wandb/sdk/internal/sender.py

```diff
@@ -484,6 +488,9 @@ def transition_state() -> None:
         elif state == defer.FLUSH_OUTPUT:
             self._output_raw_finish()
             transition_state()
+        elif state == defer.FLUSH_JOB:
```

Comment:
this order seems fine to me, but i don't know all the details of the job creation logic so I trust you here!

Reply:
Yep, I think this is correct, its after the artifacts, config and summary are flushed